src/utils/traverseHeadings.ts

```diff
@@ -1,6 +1,3 @@
-import {headerNames, propsAreEmptyByTag} from "../components/UiComponentProps";
-import featureFlagsJson from "../components/FeatureFlags/feature-flags.json";
```

Comment:
The file itself is still being used by the FeatureFlags component to render all of the feature flags, but we are removing the import from here so that it is not imported into every single page and instead if we find the FeatureFlags component we will parse it to build out the headers list for the ToC.